i18n/en.json

```diff
@@ -0,0 +1,8 @@
+{
+  "@metadata": {
+    "authors": [
+      "Dan Schultz"
+    ]
+  },
+  "smaps-desc": "SimpleMaps description goes here."
```

Comment:
Would be good to correct this before merging (I admit I don't know where this field is actually used, though)